### ISSUE TITLE
Code cleanup/comments

### ISSUE THREAD

**Issue by samcorky** [somakeit/smib#15]
general clean up of code to make more readable and add comments etc etc - no functionality changes - just re-structuring the app

**Comment by samcorky** [comment]
@sjefferson99 is this too broad of a issue?

**Comment by sjefferson99** [comment]
@sam57719 In general I would expect code cleanup to happen as and when you work on code, so fix as you go, so refactor for clarity or better layout for new function being implemented. And personally (you may disagree entirely), I refactor code to make it clear enough to remove comments and instead ensure clear docstrings for classes, methods and functions and type hinting for arguments and return types, although this seems to be quite divisive in python communities, so something we may need to agree on collab code standards at some point.

So in short, probably, maybe adjust to any specific code that is unclear?

**Comment by samcorky** [comment]
@sjefferson99 will close this down and tidy up can be done under other fixes 

I agree with the comments thing - docstrings are better. I will close this and raise the issue of adding docstrings to the existing codebase
